app/views/courses/confirmation.html.erb

```diff
@@ -56,11 +56,13 @@
                 <%= course.sorted_subjects %>
               </dd>
               <dd class="govuk-summary-list__actions">
-                <%= course_creation_change_button(
-                  "subjects",
-                  "subjects",
-                  :new_provider_recruitment_cycle_courses_subjects_path
-                ) %>
+                <% unless course.is_further_education? %>
```

Comment:
![image](https://user-images.githubusercontent.com/470137/89462367-32aba580-d765-11ea-8d0d-204cf9b87fff.png)

> 'Change' link hidden from view in subject field as there is nothing else change the subject to

The journey feels brokens on the `Basic details` tab as it exhibit `there is nothing else change the subject to` syndrome

Reply:
TBH, I think we should hide "Subjects" for FE courses as they aren't part of the course creation wizard so users cannot actually change it or does it make sense to repeat so many times. @jesseyuen, any thoughts?

Reply:
Sorry, just seeing this. @JonF1 and I had a quick chat about this yesterday but at the time I didn't realise Subjects wasn't part of the Create a Course journey, my bad!

I also agree, let's hide "Subjects" altogether @aliuk2012 @JonF1

Reply:
Ok, thanks Jesse, I will sort this out now